topsort-endpoints.yml

```diff
@@ -153,8 +163,27 @@ components:
           minimum: 0
         productId:
           type: string
-          description: The marketplace's ID for the product. It will match the ID for a product in the request's product array.
+          description: The marketplace's ID for the product. It will match the ID for a product in the request's products array.
           example: pSA0238
+        winnerType:
+          type: string
+          description: The type of the winning bid, depending on the campaign. It can be product or brand.
+          enum:
+            - brand
+            - product
+        winnerId:
+          type: string
+          description: The marketplace's ID of the winnning item, depending on the campaign.
+          example: b_Mfk15
+        eventId:
+          type: string
+          description: A topsort ID to be used when this item is interacted with.
+          example: "AKFU78:ts_brand:b_Mfk15"
```

Comment:
Please make this example opaque. The users don't need to know what the internal structure is and should not depend on that. Same for the example below.